crates/sc-consensus-subspace-rpc/src/lib.rs

```diff
@@ -249,6 +253,15 @@ where
         })
     }
 
+    fn get_best_block_number(&self) -> FutureResult<BlockNumber> {
+        let best_number = TryInto::<BlockNumber>::try_into(self.client.info().best_number)
+            .unwrap_or_else(|_| {
+                panic!("Confirmation depth K can't be converted into BlockNumber");
+            });
```

Comment:
Is it okay to panic here? Will it be handled and converted to error?

Reply:
Our blocks are always u32 and we don't expect that to ever change. This is why we panic if the number doesn't fit into u32.

This is a big ugly because code is generic, but I believe in compiled code this panic shouldn't even be present because it will effectively be conversion of u32 to u32. I don't think we need error handling here unless you feel strongly otherwise.

I just noticed error message was copy-pasted, fixed it.